### PR TITLE
TE-355 Add 'Get started' copy

### PR DESCRIPTION
**Pull request by marcdavi-es**
[Related YouTrack issue](https://youtrack.lodgify.net/issue/TE-355)

### What **one** thing does this PR do?
Adds some holding copy to the 'Get started' section. 

screen shot 2018-05-01 at 18 45 20

### CODE REVIEW

File: styleguide/docs/getStarted.md [hunk 1]
```diff
@@ -1 +1,26 @@
...
+
+### Using this documentation
+
+On the sidebar you can see an index of all the components exposed by Lodgify UI.
+
+Components are organised into categories from simple typography, elements and inputs through to oven-ready widgets to include in your site.
+
+Each component is documented with:
+* a brief description of its purpose,
+* the import path to the component in the Lodgify UI npm package,
```
Comment (davidpelayo) [inline styleguide/docs/getStarted.md]: Which is not published yet?

File: styleguide/docs/getStarted.md [hunk 2]
```diff
@@ -1 +1,26 @@
...
+* a comprehensive description of each of its props and
+* one or more examples of its usage.
+
+You can use the code editors to modify the examples live in your browser.
+
+```jsx
+<Paragraph>Cool, huh?</Paragraph>
+<Button onClick={() => console.log('Totally')}>Click me</Button>
```
Comment (davidpelayo) [inline styleguide/docs/getStarted.md]: Better to use an `alert` maybe?

File: styleguide/docs/getStarted.md (hunk 1 above)
Comment (marcdavi-es) [reply styleguide/docs/getStarted.md]: Coming soon 👯

File: styleguide/docs/getStarted.md (hunk 2 above)
Comment (marcdavi-es) [reply styleguide/docs/getStarted.md]: I loathe `alert`

**Pull request merged**